source/Cosmos.System2/Graphics/Bitmap.cs

```diff
@@ -52,107 +52,121 @@ public Bitmap(uint Width, uint Height, byte[] pixelData, ColorDepth colorDepth)
 
         private void CreateBitmap(Stream stream)
         {
-            #region BMP Header
-
-            Byte[] _int = new byte[4];
-            Byte[] _short = new byte[2];
-            //Assume that we are using the BMP (Windows) header format
-            //I am using http://www.fastgraph.com/help/bmp_header_format.html
-            //and https://upload.wikimedia.org/wikipedia/commons/c/c4/BMPfileFormat.png?1519566101894as as reference
-            stream.Position = 10;
-            //read header - bytes 10 -> 14 is the offset of the bitmap image data
-            stream.Read(_int, 0, 4);
-            uint pixelTableOffset = (uint)BitConverter.ToInt32(_int, 0);
-
-            //now reading size of BITMAPINFOHEADER should be 40 - bytes 14 -> 18
-            stream.Read(_int, 0, 4);
-            uint infoHeaderSize = (uint)BitConverter.ToInt32(_int, 0);
-            if (infoHeaderSize != 40)
+            using (stream)
```

Comment:
You are already using "using" in the Ctor that takes a file path, so it is not need to write it here.